tools/offline_ota_image_builder/README.md

```diff
@@ -0,0 +1,173 @@
+# Offline OTA image builder
+
+`tools.offline_ota_image_builder` is a helper package for building offline OTA image from several input OTA images. Built image can be used as external cache source for otaproxy, or used by full offline OTA helper util to trigger fully offline OTA update.
+
+**NOTE**: full offline OTA helper util is not yet implemented.
+
+It provides the following features:
+
+1. build offline OTA image with several input OTA images,
+2. export built image as tar archive,
+3. prepare and export built image onto specific block device as external cache source dev.
+
+Please check documentation at [OTA cache mechanism design for otaproxy](https://tier4.atlassian.net/l/cp/yzmnPx9T) for more details.
+
+## Expected image layout for input OTA images
+
+Please refere to [ota-metadata](https://github.com/tier4/ota-metadata) repo for OTA image layout specification.
+This tool is implemented based on the image layout implemented in [ota-metadata@a711013](https://github.com/tier4/ota-metadata/commit/a711013).
```

Comment:
What do you mean by `This tool is implemented based on the image layout`?
Is this means `This tool is implemented based on the image layout as a input OTA image`?

Reply:
Ah what I mean here is this tool expecting the input OTA images has the same image layout with the image generated by `ota-metadata`.

What about the following rephrased sentence:
This package currently only recognizes the OTA image generated by [ota-metadata@a711013](https://github.com/tier4/ota-metadata/commit/a711013) script, or the image layout compatible with the `ota-metadata` specification.

Reply:
> What about the following rephrased sentence:

Good!